tests/core/test_playback.py

```diff
@@ -124,6 +125,22 @@ def test_play_skips_to_next_on_unplayable_track(self):
         self.assertEqual(
             self.core.playback.current_tl_track, self.tl_tracks[3])
 
+    def test_play_skips_to_next_on_unplayable_track(self):
+        """Checks that we handle change track failing."""
+        self.playback2.change_track().get.return_value = False
```

Comment:
Instead of actually calling the method on the first mock and increase it's call counts, use return_value to get hold of the inner mock.

```
self.playback2.change_track.return_value.get.return_value = False
```